Sources/SourceControl/GitRepository.swift

```diff
@@ -208,7 +209,7 @@ public class GitRepository: Repository, WorkingCheckout {
             /// The name of the object.
             let name: String
         }
-        
+
```

Comment:
Yeah - sorry about that - didn't intend the whitespace changes, but did a quick save & edit with VM and it cleans extra whitespace from lines from previous settings. I'll undo that with a future update.